src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
+
+  for (let i = 0; i < propertyValues.length; i++) {
+    current = propertyValues[i];
+
+    for (let j = i + 1; j < propertyValues.length; j++) {
+      if (current === propertyValues[j]) {
+        return null;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    inverse[robot[key]] = key;
```

Comment:
Instead of using a separate loop to create the inverse object, you can combine the duplicate check and object construction into a single loop. This will improve the efficiency of your function.